offchain/indexer/tests/integration.rs

```diff
@@ -38,7 +38,7 @@ async fn indexer_inserts_inputs() {
     const N: u64 = 3;
     let mut inputs = vec![];
     for i in 0..N {
-        let input = state.produce_input_in_broker(i).await;
+        let input = state.add_input_in_broker(i).await;
```

Comment:
Perhaps replacing `in_broker` with `to_stream` in all similar named functions would make more sense.
```suggestion
        let input = state.add_input_to_stream(i).await;
```